src/eval-frame/actions/fetch-cell-eval-actions.js

```diff
@@ -115,54 +110,49 @@ export async function handleFetch(fetchInfo) {
 }
 
 export async function evaluateFetchText(fetchText, evalId) {
-  const outputHistoryId = generateRandomId();
   const fetches = parseFetchCell(fetchText);
   const syntaxErrors = fetches.filter(fetchInfo => fetchInfo.parsed.error);
   if (syntaxErrors.length) {
-    sendActionToEditor(
-      addToConsoleHistory({
-        historyType: "FETCH_CELL_INFO",
-        value: syntaxErrors.map(fetchProgressInitialStrings),
-        historyId: outputHistoryId,
-        level: "ERROR"
-      })
-    );
+    addConsoleEntryInEditor({
+      historyType: "FETCH_CELL_INFO",
+      content: syntaxErrors
+        .map(fetchProgressInitialStrings)
+        .map(t => t.text)
+        .join(""),
```

Comment:
just use `historyItem.content` strings rather than array of objects. strings behave better with shallow compare and serialize easier everywhere, and this was just displayed in a <pre> anyway